### PR TITLE
Implemented dynacli script for initializing CLI entrypoint starter file

### LINKED ISSUE #73: [FEATURE] Add CLI bootstrap script
**Is your feature request related to a problem? Please describe.**

From starting point having some kind of bootstrapper will be beneficial.

`dynacli startproject .` -> will create CLI entrypoint with some common case code.

This will ease the process of getting started.

* It should be installed via pip(alongside dynacli)
* It should have zero dependency to 3rd party packages
* It should not remove, modify, rename, change etc any existing code base
* It should have 2 modes - default and interactive:
     - Interactive mode will ask for path, CLI entrypoint name, description and version.
     - default mode will create it in given path with predefined code.

### PR DESCRIPTION
**Pull request by ShahriyarR**
# Change Summary

## Description

`dynacli init awesome .` -> it will create awesome executable in the current directory or specified directory

Fixes #73 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/dynacli/bootstrap/init.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Initialize the CLI entrypoint in given path
+
+    Args:
+        name (str): name of the new CLI tool
+        path (str): The path for CLI entrypoint
+
+    Return: None
+    """
+    wd = os.path.dirname(os.path.realpath(__file__))
+    cwd = path if path != "." else os.path.curdir
```
Comment (asterkin) [inline src/dynacli/bootstrap/init.py:17]: Problematic user experience, you enforce me to type dynacli initi mycli . most of the time. The alternative would be to define **kwargs and recognise path=. In the future, you could support additional optional parameters such as description, version, root, etc.

File: src/dynacli/bootstrap/init.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    try:
+        shutil.copyfile(f"{wd}/_sample_cli", f"{cwd}/{name}")
```
Comment (asterkin) [inline src/dynacli/bootstrap/init.py:19]: Not sure about this. My expectations would be that by typing dynacli init awesome you would create awesome folder with awesome script undneath, with path/root if defined and even a basic install script. What is done here is a bit too little. But considering time pressures, you may stage developing these features.

File: src/dynacli/bootstrap/_cli.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```
Comment (asterkin) [inline src/dynacli/bootstrap/_cli.py:1]: I did not understand why do you need this file?

File: src/dynacli/bootstrap/init.py (hunk 1 above)
Comment (asterkin) [reply src/dynacli/bootstrap/init.py:17]: If you want a quick release, we may just drop the path argument and defer more advanced features to next release.

**Review by asterkin — CHANGES_REQUESTED** [review]

File: src/dynacli/bootstrap/_cli.py (hunk 3 above)
Comment (ShahriyarR) [reply src/dynacli/bootstrap/_cli.py:1]: This is for entry_point of the package, when you install dynacli from pip - this file's main function acts as entrypoint script.

File: src/dynacli/bootstrap/init.py (hunk 1 above)
Comment (ShahriyarR) [reply src/dynacli/bootstrap/init.py:17]: Let's release it now, as we need to release with python 3.7 support + medium article.
Mostly this feature just created for our medium article - then when we need something serious we can improve it further,

Comment (ShahriyarR) [reply src/dynacli/bootstrap/init.py]: Switched to **kwargs and accepting path= -> if not path= provided then will go with current directory

**Pull request merged**